examples/workflow-theia/server/download.ts

```diff
@@ -20,9 +20,11 @@ const downloadDir = resolve(join(__dirname));
 const mavenRepository = "https://oss.sonatype.org/content/repositories/snapshots/";
 const groupId = "org.eclipse.glsp.example";
 const artifactId = "org.eclipse.glsp.example.workflow";
-const version = "0.8.0";
+const version = "0.9.0";
 const classifier = "glsp";
 
-console.log("Download latest version of the Workflow Example Java Server from the maven repository.");
+console.log("Downloading latest version of the Workflow Example Java Server from the maven repository...");
 download({ groupId, artifactId, version, classifier, isSnapShot: true }, downloadDir, mavenRepository)
-    .then(() => console.log("Download completed."));
+    .then(() => console.log("Download completed. Start the server (Theia and GLSP server) using these commands: \ncd examples/browser-app\nyarn start\n\n"
```

Comment:
```suggestion
    .then(() => console.log("Download completed. Start the Theia back-end using these commands: \ncd examples/browser-app\nyarn start\n\n"
```

Reply:
Because these commands actually only start the Theia backend and the GLSP server is started automatically by the Theia backend.